backend/src/main/java/ca/mcgill/purposeful/controller/CollaborationRequestController.java

```diff
@@ -46,4 +49,21 @@ public ResponseEntity<CollaborationRequestDTO> sendCollaborationRequest(
                 email, ideaId, message, additionalContact)),
         HttpStatus.OK);
   }
+
+  /**
+   * This method returns all collaboration requests linked to a given idea
+   *
+   * @param ideaId - the id of the idea
+   * @return a list of collaboration request DTOs linked to the idea
+   * @throws GlobalException if the idea does not exist
+   */
+  @GetMapping("/{ideaId}")
+  @PreAuthorize("hasAuthority('User')")
+  public ResponseEntity<List<CollaborationRequestDTO>> getCollaborationRequestsByIdea(
```

Comment:
Can we add a check that the person asking for the requests is the owner of the idea? Thanks